src/layouts/SiteDashboard/SiteDashboard.tsx

```diff
@@ -193,9 +200,67 @@ export const SiteDashboard = (): JSX.Element => {
           {/* Right column */}
           <Box w="40%">
             <VStack spacing="1.25rem">
+              {/* Site launch display card */}
+              {isUserUsingSiteLaunchFeature(siteName) &&
+                siteLaunchStatus !== "LAUNCHED" && (
```

Comment:
Actually i feel that this might be better off as a stand-alone component (in this file) that's used here. 

why this is so is because this chunk relies solely on a flag + context variable that the rest of the file is ignorant of (less the default portion), so having the site launch stuff exposed here might not make the most sense, especially because the text etc is also dependent on the context.

Reply:
ok refactored this bit